vcpkg.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "libleidenalg",
+  "version": "0.9.0",
+  "maintainers": "Andrew Robbins <andrew@robbinsa.me>",
```

Comment:
Depending on what this entails, I'm not sure whether you contact details or mine would be more relevant here? Would this refer to the port in `vcpkg` (which I'm happy with you maintaining), or would this refer to the code in this repository (which in principle I would be maintaining)?

Reply:
This one should be you, I think.